src/_utils/rideAxis/story.tsx

```diff
@@ -7,7 +7,7 @@ import { color, font } from '../branding'
 import { RideAxis } from './index'
 import readme from './specifications/index.md'
 
-const stories = storiesOf('Internal|RideAxis', module)
+const stories = storiesOf('Widgets|RideAxis', module)
```

Comment:
this should not be here ?

Reply:
I'm moving the widgets by group of function like "Design Tokens", `RideAxis` where all alone so I move it back to Widgets, I can do it on other PR, though.. no problem